src/kakarot/kakarot.cairo

```diff
@@ -204,12 +91,41 @@ func deploy_externally_owned_account{
     return Kakarot.deploy_externally_owned_account(evm_address);
 }
 
-// @notice Compute the starknet address of a contract given its EVM address
-// @param evm_address The EVM address of the contract
-// @return contract_address The starknet address of the contract
+// @notice The eth_call function as described in the spec, see https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_call
```

Comment:
appreciate linking out to the spec!

one short little comment that describes the difference of how eth_call differs from invoke might be useful for people new to blockchains/eth/starknet would be good here also.

the simplest way to put it is (maybe) its an evaluation of bytecode that doesn't update any state.